MultiPartyExample/LocalParticipantView.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  LocalParticipantView.swift
+//  MultiPartyExample
+//
+//  Created by Ryan Payne on 4/17/19.
+//  Copyright © 2019 Twilio, Inc. All rights reserved.
+//
+
+import UIKit
+import TwilioVideo
+
+@IBDesignable
+class LocalParticipantView: UIView {
+
+    @IBOutlet var contentView: UIView!
+    @IBOutlet weak var videoView: TVIVideoView!
+    @IBOutlet weak var noVideoImage: UIImageView!
+    @IBOutlet weak var audioIndicator: UIImageView!
+    @IBOutlet weak var networkQualityLevelContainerView: UIView!
+    @IBOutlet weak var networkQualityLevelIndicator: UIImageView!
```

Comment:
Based upon the current design, the container should be positioned such that it is entirely contained within the video. It looks odd on an iPhone X.

![IMG_0041](https://user-images.githubusercontent.com/1302577/56387399-0f0fc500-61d9-11e9-8c8a-749978041607.PNG)

Reply:
Will double check the containerView's layout constraints on the X. This should be taking the entire space.